fed/api.py

```diff
@@ -52,6 +53,7 @@ def init(
     cross_silo_send_max_retries: int = None,
     cross_silo_serializing_allowed_list: Dict = None,
     exit_on_failure_cross_silo_sending: bool = False,
+    grpc_max_size_in_bytes: int = None,
```

Comment:
`grpc_max_size_in_bytes` -> `cross_silo_message_max_size_in_bytes` could be more explicit?

Reply:
Sounds good to me@